ciphers/vigenera_cipher_dictionary_hacker.py

```diff
@@ -0,0 +1,43 @@
+import detectEnglish, vigenereCipher, pyperclip
+
+
+def main():
+    ciphertext = "Tzx isnz eccjxkg nfq lol mys bbqq I lxcz"
+    hackedMessage = hackVigenere(ciphertext)
+
+    if hackedMessage != None:
+        print("Copying hacked message to clipboard:")
+        print(hackedMessage)
+        pyperclip.copy(hackedMessage)
+    else:
+        print("Failed to hack encryption.")
+
+
+def hackVigenere(ciphertext: str) -> str:
+    """
+    >>> hackVigenere("Tzx isnz eccjxkg nfq lol mys bbqq I lxcz")
+    'Hjsg sfaty shsj hjasg vzfwt jnal'
+    
+    
+    Return the decrypted text
+    """
+    with open("dictionary.txt") as in_file:
+        words = in_file.readlines()
+    for word in words:
+        word = word.strip()
+        decryptedText = vigenereCipher.decryptMessage(word, ciphertext)
+        if detectEnglish.isEnglish(decryptedText, wordPercentage=40):
+            # Check with user to see if the decrypted key has been found.
+            print()
+            print("Possible encryption break:")
+            print("Key " + str(word) + ": " + decryptedText[:100])
+            print()
+            print("Enter D for done, or just press Enter to continue breaking:")
+            response = input("> ")
```

Comment:
As discussed in CONTRIBUTING.md, algorithmic functions should not print() and input() will hang our automated testing.